chart-parts/eirini-namespace.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if and .Values.enable.eirini }}
```

Comment:
You can drop `and` here (unless it's doing something special that I'm not aware of).

Reply:
No, it is not supposed to be there. The line used to have a more complex condition, but was later refactored, but the `and` stayed behind. Will fix.